tests/_graphical_login.pm

```diff
@@ -20,10 +20,20 @@ sub run {
 
     # Wait for the password entry to appear.
     assert_screen('gdm_login_password', 10);
+    sleep(8);
+
+    # FIXME: Click out of and back into the password box, in case it’s not
+    # focused properly. This should really be investigated in gdm itself.
```

Comment:
Does this (seem to) help? :sob:

Reply:
The whole commit does seem to help, but I haven’t done any measurements, and definitely haven’t broken down the individual changes in the commit to see how well each of them help. It’s been more a case of “throw everything I can think of at the problem and hope that placates it”.

I am sorry.